mindjunkies/payments/tests/test_views.py

```
@@ -1,115 +1,158 @@
-from django.contrib.messages import get_messages
 from django.urls import reverse
 from model_bakery import baker
+from unittest.mock import patch
+from django.contrib.messages import get_messages
 
 from mindjunkies.accounts.models import User
 from mindjunkies.courses.models import Course, Enrollment
-from mindjunkies.payments.models import PaymentGateway, Transaction
-from django.utils.timezone import make_aware
-from datetime import datetime
+from mindjunkies.payments.models import Transaction, Balance, BalanceHistory
```

Ruff (0.8.2)

`mindjunkies.payments.models.Balance` imported but unused

Remove unused import: `mindjunkies.payments.models.Balance`

(F401)